api/Document.json

```diff
@@ -5892,7 +5892,7 @@
               "version_added": "33"
             },
             "edge": {
-              "version_added": true
+              "version_added": "18"
```

Comment:
`document.onvisibilitychange` confirmed undefined in Edge 17 and null in Edge 18.